code/modules/antagonists/ert/ert.dm

```diff
@@ -172,6 +172,10 @@
 	missiondesc += "<BR><B>Your Mission</B> : [ert_team.mission.explanation_text]"
 	to_chat(owner,missiondesc)
 
+/datum/antagonist/ert/amber/greet()
+	. = ..()
+	to_chat(owner,"<BR><B>Your Mission</B> : You also have a combat knife inside your boots.")
```

Comment:
```suggestion
	to_chat(owner,"You also have a combat knife inside your boots")
```
otherwise it says Your Mission : You also have a combat knife inside your boots.

Reply:
Good catch